MobileBuy/buy/src/main/java/com/shopify/buy/model/Address.java

```diff
@@ -62,12 +62,29 @@
 
     private String zip;
 
+    @SerializedName("default")
+    private Boolean defaultAddress;
```

Comment:
hmm, this is how we handle it in some other classes as well.  We don't want to use the primitive because we don't want to serialize/deserialize if the JSON does not have it set.
But in practice this value is always set on the JSON from the server so we haven't seen any issues.
Is will put a ticket in the backlog, we should be null checking in all the places we get the Boolean and convert to boolean just in case.